application/backend/src/main/java/io/typerefinery/websight/models/components/widgets/Ticker.java

```diff
@@ -106,6 +96,26 @@ protected void init() {
     @Default(values = "")
     public String flowapi_topic;
 
+    @Inject
+    @Getter
+    @Default(values = "col-3")
```

Comment:
Default should be empty and that should mean the component takes up available horizontal space?

Reply:
Yeah updated to col, this will take up the remaining spaces by default.